shellcheck/shellcheck.yml

```diff
@@ -0,0 +1,24 @@
+name: ShellCheck
+
+on:
+  - push
+  - pull_request
```

Comment:
We usually explicitly list branches.

Reply:
Updated to make that for pull requests only and with an explicit list